### PR TITLE
Hotfix for protobuf==3.21.9 in CI

### PR DESCRIPTION
**Pull request by gkrivor**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: conanfile.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@
 ade/0.1.2a
 onetbb/[>=2021.2.1]
 pugixml/[>=1.10]
-protobuf/[>=3.20.3]
+protobuf/3.21.9
```
Comment (rkazants) [inline conanfile.txt:5]: Let us have it to be aligned with constraint files where we have: `protobuf>=3.18.1,<4.0.0`

**Comment by gkrivor** [comment]
Actually, I still misunderstand why we get misalignment between versions. @ilya-lavrenov did you find exact root cause?

File: conanfile.txt [hunk 2]
```diff
@@ -2,7 +2,7 @@
 ade/0.1.2a
 onetbb/[>=2021.2.1]
 pugixml/[>=1.10]
-protobuf/[>=3.20.3]
```
[the remark this answers is not in the document]
Comment (ilya-lavrenov) [reply conanfile.txt:5]: `tool_requires` are used install `protoc` to generate .pb.cc and pb.h files, while `requires` is responsible for libraries (e.g. `libprotobuf` or `libprotobuf-lite`). 
Protobuf requires that `protoc` and `libprotobuf` must have the same versions.

**Comment by ilya-lavrenov** [comment]
Fixed by https://github.com/openvinotoolkit/openvino/pull/18351

**Pull request closed without merging**